.github/workflows/tests.yaml

```diff
@@ -20,9 +20,25 @@ jobs:
     - name: Install pip dependencies
       run: |
         pip install cython numpy  # needed for pycocotools
-        pip install .[datasets,tests,train]
+        pip install .[datasets,tests]
     - name: Run mypy checks
       run: mypy .
+  datasets:
```

Comment:
Identical to our other tests but without the optional dataset dependencies to ensure that we are skipping them properly.